src/platform/silabs/EFR32/BLEManagerImpl.cpp

```diff
@@ -157,6 +157,9 @@ CHIP_ERROR BLEManagerImpl::_SetAdvertisingEnabled(bool val)
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     VerifyOrExit(mServiceMode != ConnectivityManager::kCHIPoBLEServiceMode_NotSupported, err = CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+    DeviceLayer::ConnectivityMgr().RequestSEDActiveMode(val);
```

Comment:
This is already handled in the common code - `CommissioningWindowManager`. If something is not working it's probably better to investigate existing code instead of adding another behavior triggered by the same event.

Reply:
Well this is not handled at all in the `CommissioningWindowManager` There is an `if` preventing the `RequestSEDActiveMode` when BLE is enabled. In the `CommissioningWindowManager` what seems to be handled is the use case for OnNetworkCommissioning and not the BLE commissioning. 

This problem happens with long sleep time and  I wasn't sure if the problem was specific to us or common to everyone. If you can confirm that you're having trouble commissioning with BLE and a long sleep time ( in the minutes range, let say > 5min) I'll gladly modify the `CommissioningWindowManager` to have a common solution in place.

Reply:
Honestly, we haven't tested that long sleep time, but I'm sure everyone will have the same problem if the idle interval is very long so IMHO it would be beneficial to fix it in `CommissioningWindowManager`. Btw. with this code you have, doesn't `SetAdvertisingEnabled(false)` cause switching back to the idle mode? My understanding is that the advertising is stopped on PASE session establishment,  so if you have a problem with receiving Sigma1 I'm not sure why it helps. Could you help me understand?